### ISSUE TITLE
Generated favicons, placed in site but report still says some are missing/corrupted

### ISSUE THREAD

**Issue by u01jmg3** [RealFaviconGenerator/realfavicongenerator#63]
### Issues
- The favicon is not well-designed for Android.
  - _It exists and was generated by RFG_
- Some picture(s) are missing in favicon.ico:
  - 32x32 picture _(It exists and was generated by RFG)_
  - 48x48 picture _(It exists and was generated by RFG)_
- Some PNG favicon(s) could not be found or are corrupted.
  - _Not sure which one(s)_
- Some recommended pictures for Windows 8 / IE 11 are missing:
  - 70x70 picture _(does exist)_
  - 150x150 picture _(does exist)_
  - 310x150 picture _(does exist)_
  - 310x310 picture _(does exist)_
### Other Issues
- I have noticed that the `favicon-196x196.png` generated by RFG has been based on the `apple-touch-icon.png` I provided instead of the `260x260.png` I provided as my main favicon

![example](https://cloud.githubusercontent.com/assets/1266205/3969577/0b31bf8e-27c2-11e4-853a-8798d6fa6432.png)
- I have noticed that the `link rel` code for adding to `<head>` doesn't mention `apple-touch-icon-precomposed.png` or `apple-touch-icon.png` even though it's in a custom folder (`images/touch/`)
- I have noticed that when initially running the report (on a site with just a favicon and nothing else) the report does not mention the 57x57 apple-touch-icon yet generates one when running RFG; is this missing from the report summary?
### `browserconfig.xml`
- https://homepages.abdn.ac.uk/jonathan.goode/pages/test/browserconfig.xml
### Structure

```
│   favicon.ico
│   browserconfig.xml
│   index.html
│
└───images
    └───touch
            apple-touch-icon-114x114.png
            apple-touch-icon-120x120.png
            apple-touch-icon-144x144.png
            apple-touch-icon-152x152.png
            apple-touch-icon-57x57.png
            apple-touch-icon-60x60.png
            apple-touch-icon-72x72.png
            apple-touch-icon-76x76.png
            apple-touch-icon-precomposed.png
            apple-touch-icon.png
            favicon-160x160.png
            favicon-16x16.png
            favicon-196x196.png
            favicon-32x32.png
            favicon-96x96.png
            mstile-144x144.png
            mstile-150x150.png
            mstile-310x150.png
            mstile-310x310.png
            mstile-70x70.png
```
- http://realfavicongenerator.net/favicon_checker?site=homepages.abdn.ac.uk%2Fjonathan.goode%2Fpages%2Ftest

![screenshot](https://cloud.githubusercontent.com/assets/1266205/3969282/e17a4326-27be-11e4-8842-5622912f2b08.png)

**Comment by phbernard** [comment]
Regarding the report: yep, there is a bug with the checker. I think this is due to the combination of two things: the page you test is in a subdirectory (ie. not the root of the web site, eg. http://homepages.abdn.ac.uk/ ) and the path of the icons is relative. That should work of course. I expect to investigate that this week.

Regarding the 196x196 icon: [there is a reason for this](http://realfavicongenerator.net/faq#why_196x196_different)... [but not a good one](https://github.com/RealFaviconGenerator/realfavicongenerator/issues/26). That should be changed quite soon.

That's right, [some Apple touch icons are not declared](http://realfavicongenerator.net/faq#why_precomposed). This is mostly a safety measure.

Regarding the 57x57 icon: the report is a bit messy due to the issue you found. I'll check this once the main bug is fixed.

**Comment by phbernard** [comment]
Fixed! Please visit http://realfavicongenerator.net/favicon_checker?site=homepages.abdn.ac.uk%2Fjonathan.goode%2Fpages%2Ftest again.

**Comment by phbernard** [comment]
I think this issue can be closed. Please reopen it if necessary.

**Comment by u01jmg3** [comment]
Sorry @phbernard for my slow reply - I didn't seem to get a notification from GitHub that you had replied

Looks like things are working much better but I am still facing two issues - one new and one leftover from last time.
1. The issue left over from last time is that I am still seeing _Some recommended pictures for Windows 8 / IE 11 are missing_ even though they exist and are correctly defined in [browserconfig.xml](http://homepages.abdn.ac.uk/jonathan.goode/pages/test/browserconfig.xml)
   - Both checking and unchecking the checkbox _My favicon files are not in the root directory of my web site._ has no bearing on this error - it still occurs
2. The new issue is that using these [settings](https://cloud.githubusercontent.com/assets/1266205/5236441/05655b96-7831-11e4-8010-a6121badb4d1.png) all the iOS icons are generated using the image I selected for Windows 8 rather than the image I selected for iOS.

![issue](https://cloud.githubusercontent.com/assets/1266205/5236485/6d6c5220-7832-11e4-8edb-f18981f70b30.png)
